rust/sealevel/environments/mainnet3/warp-routes/eclipseusdc/token-config.json

```diff
@@ -1,11 +1,4 @@
 {
-  "solanamainnet": {
```

Comment:
does the new one need to be added here? or just bc we only connect eclipse <> ethereum for usdc, this is fine

Reply:
added back 🫡